README.md

```diff
@@ -18,6 +18,17 @@ named `welcome-demo`. `cd` to it.
    git push origin add-<yourname>-fav-num
    ```
 
+   (Hint: For a one-line log message like in this Welcome Demo, `git
+   commit -m` is convenient.  However, you should often write longer
+   log messages, with an initial subject line and a descriptive body.
+   In that case, omit the `-m` option.  Git will open up a text editor
+   for you to enter your log message, so that you can edit multi-line
+   messages with access to convenient editing commands like filling
+   paragraphs.  (You can configure your account to use a different
+   editor if you don't like the default editor.  To submit the log
```

Comment:
Is the close parenthesis missing here?

Reply:
I agree that maybe the close paren should be moved up to after "default editor".

Reply:
Actually, it's a missing close paren. Do you think it's too confusing to have an embedded parenthetical expression?